CMakeLists.txt

```diff
@@ -217,14 +218,61 @@ set(CPACK_PACKAGE_VERSION_PATCH ${VERSION_PATCH})
 set(CPACK_PACKAGE_VENDOR "celeron55")
 set(CPACK_PACKAGE_CONTACT "Perttu Ahola <celeron55@gmail.com>")
 
+include(CPackComponent)
+
+cpack_add_component(SUBGAME_MINETEST_GAME
+	DISPLAY_NAME "Minetest Game"
+	DESCRIPTION "The official subgame for the Minetest engine, that can easily extended by mods."
+	GROUP "Subgames"
+)
+
+cpack_add_component(SUBGAME_MINIMAL
+	DISPLAY_NAME "Minimal development test"
+	DESCRIPTION "A minimal subgame helping to develop the engine."
+	DISABLED #DISABLED does not mean it is disabled, it means its not selected on default!
+	GROUP "Subgames"
+)
+
+cpack_add_component_group(Subgames
+	DESCRIPTION "Games for the Minetest engine."
+)
+
 if(WIN32)
-	if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-		set(CPACK_PACKAGE_FILE_NAME "${PROJECT_NAME}-${VERSION_STRING}-win64")
+	# Include all dynamically linked runtime libaries
+	include(InstallRequiredSystemLibraries)
+
+	if(RUN_IN_PLACE)
```

Comment:
> Well I don't know. :laughing: How did you generate your 7zip packages you uploaded here? You could probably do the same only as zip format.

I use `make package` but that's not the point.
The problem I have here is that you are willfully breaking package creation when cross-compiling for Windows with RUN_IN_PLACE=0

(also since you didn't seem to understand: the WiX toolset does <b>not</b> work on Linux, like not at all)


> The problem is to tell the kiddies that "zip package 1" finds its mods and conf in a different place then "zip package 2" :thinking:

That's true I guess.
But IMO trying to protect devs from building broken packages is not useful, previously the build system did not prevent building RUN_IN_PLACE=0 ZIP packages either.